### PR TITLE
fix: wrong `dgram` compatibility info

### PR DESCRIPTION
**Pull request by hey-ab**
That PR fix wrong compatibility info about `node:dgram`. In fact, the realization is quite partial and most popular libraries using `dgram` not working in Deno.
Proof: https://github.com/denoland/deno/issues/18324#issuecomment-1478034481
Some issues with `dgram`: https://github.com/denoland/deno/issues?q=dgram

### CODE REVIEW

**Comment by hey-ab** [comment]
My mistake, see https://github.com/denoland/deno/issues/18324#issuecomment-1737141836

File: runtime/manual/node/compatibility.mdx [hunk 1]
```diff
@@ -140,11 +140,25 @@ which modules you need by
   <summary>
     <code>node:dgram</code>
     <div style={{ float: "right" }}>
-      <span>✅</span>
+      <span>ℹ️</span>
     </div>
   </summary>
   <p>
-    Fully supported.
+    Much of the dgram module internals is yet to be implemented.
```
Comment (kwhinnery) [inline runtime/manual/node/compatibility.mdx:147]:
```suggestion
```

File: runtime/manual/node/compatibility.mdx [hunk 2]
```diff
@@ -140,11 +140,25 @@ which modules you need by
...
+    Some <code>dgram.Socket</code> instance methods is non-functional stubs:
```
Comment (kwhinnery) [inline runtime/manual/node/compatibility.mdx:148]:
```suggestion
    Some <code>dgram.Socket</code> instance methods are non-functional stubs:
```

**Review by kwhinnery — CHANGES_REQUESTED** [review]

**Comment by kwhinnery** [comment]
Thanks for this @hey-ab - made a few suggestions. I will go ahead and commit them unless you object, and merge this in the next ~day or so unless I hear from you. Thanks again!

**Pull request merged**